### PR TITLE
Initial work

### PR DESCRIPTION
**Pull request by kcartier-wri**
Converting Actions workflow to use Conda .

### CODE REVIEW

**Comment by kcartier-wri** [comment]
It took some fiddling around, but I got all workflow to pass using Conda. I kept the pip version, but commented it out so it doesn't launch a separate container.

**Review by jterry64 — APPROVED** [review]
Awesome, thanks!

File: .github/workflows/dev_ci_cd_conda.yml [hunk 1]
```diff
@@ -9,29 +9,31 @@ permissions:
...
       run: |
         sudo apt update
         sudo apt install -y gdal-bin libgdal-dev
-    - name: Install Packages
+    - name: Install other packages
       run: |
-        python -m pip install --upgrade pip
-        pip install -r .github/requirements.txt
-        pip install GDAL==`gdal-config --version`
+        conda install pytest --yes
```
Comment (chrowe) [inline .github/workflows/dev_ci_cd_conda.yml:33]: Isn't pytest installed in `cities-cif`? Why do we need it installed here?

File: .github/environment.yml
Comment (chrowe) [inline .github/environment.yml:1]: Is there a reason to use a separate file for Actions vs just using the main project one?

**Comment by kcartier-wri** [comment]
Two good points and better solution. Implemented in https://github.com/wri/cities-cif/pull/79 .

**Pull request merged**